src/main/java/zmaster587/advancedRocketry/block/BlockAdvancedBipropellantRocketMotor.java

```diff
@@ -4,7 +4,7 @@
 import net.minecraft.util.math.BlockPos;
 import net.minecraft.world.World;
 
-public class BlockAdvancedBipropellantRocketMotor extends BlockRocketMotor {
+public class BlockAdvancedBipropellantRocketMotor extends BlockBipropellantRocketMotor {
```

Comment:
This is the change to class inheritance that I mentioned in the commit and merge messages.

Reply:
Don't think that's going to be a problem as long as it doesn't mess with the logic I wrote for it.